docs/source/technical_tutorials/visualization/visualize_lonboard.ipynb

```diff
@@ -0,0 +1,574 @@
+{
```

Comment:
<p>Can simplify to - </p>

<pre class="ql-syntax" spellcheck="false">layer = ScatterplotLayer.from_geopandas(gdf,
&nbsp;&nbsp;get_fill_color=[0, 255, 0],&nbsp;
&nbsp;&nbsp;radius_min_pixels=4
)
Map(layer)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>